src/sass/components/_section--brands.scss

```diff
@@ -0,0 +1,53 @@
+.section--brands {
+  margin: 30px 0 60px 0;
+  height: 140px;
+  .container {
+    border-top: 1px solid #ddd;
+    border-bottom: 1px solid #ddd;
+  .logos {
+    position: relative;
+    width: 85%;
+    margin: 0 auto;
+    height: 140px;
+    padding: 10px 0;
+    .col-2 {
+      padding: 0 4px;
+    }
+    .logoBox {
+      border: 1px solid #ddd;
+      padding: 10px;
+      height: 90px;
+      position: relative;
+      top: calc(50% - 45px);
+      text-align: center;
+      display: flex;
+      align-items: center;
+      .logo {
+        height: auto;
+        max-height: 100%;
+        width: auto;
+        max-width: 100%;
+      }
+    }
+    .arrow {
+      background: $footer-claim-bg;
+      color: white;
```

Comment:
kolory w HEX

Reply:
O ile w przypadku koloru białego to nie ma aż takiego znaczenia tak np `yellow` różnie potrafi się zachowywać w różnych przeglądarkach i na różnych urządzeniach. Lepiej używać hex :)